examples/Querying Chromeleon Data.ipynb

```diff
@@ -0,0 +1,276 @@
+   "metadata": {},
+   "source": [
+    "## Configure Python Imports\n",
+    "\n",
+    "In order to run our Notebook, we will need the following libraries:\n",
+    "\n",
+    "- requests: This library is used to make HTTP Requests to the TDP API\n",
+    "- json: Allows us to manipulate files as JSON Objects\n",
+    "- pandas: This is a very useful library for storing data in tabular structures\n",
+    "- numpy: Open Source Framework for mathematical computation\n",
+    "- matplotlib: Library for creating visualizations of your data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b9307f6e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import requests, json, pandas as pd, numpy as np, matplotlib.pyplot as plt\n",
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4a9140e4",
+   "metadata": {},
+   "source": [
+    "## Configure Connection Variables\n",
+    "\n",
+    "Create and store information on how to connect to the TDP API"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "95c7eaac",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "api_root = \"\"\n",
+    "search_url = api_root + \"/v1/datalake/searchEql\"\n",
+    "retrieve_file = api_root + \"/v1/datalake/retrieve\"\n",
+    "print(search_url)\n",
+    "print(retrieve_file)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b9abc644",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "org_slug = \"\"\n",
+    "user_token = \"\"\n",
+    "headers = {\"x-org-slug\": org_slug, \"ts-auth-token\": user_token}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "87f6e7d2",
+   "metadata": {},
+   "source": [
+    "## Create Query to Search for All Injections in a Project\n",
+    "\n",
+    "The TDP API uses ElasticSearch for indexing the Chromeleon Content. This powerful tool allows advanced searching against the data to find the appropriate information based upon your use case(s).\n",
+    "\n",
+    "In this scenario, we are creating a query to find all of the Chromeleon Data for a specific sequence/folder in a Data Vault.\n",
+    "\n",
+    "If you want to find all the possible sequences/folders, Navigate in the TetraScience Scientific Data Cloud to the Search File tab. Click on Browse, next to List in the top right. Then navigate down the folder list to find see the sequences.\n",
+    "\n",
+    "Here we grab all the injections from the project under \"SEQUENCE_NAME.seq\" by searching for all files with a file path that contains the sequence/project/directory name. Change the sequence name to be relevant to your data.\n",
+    "\n",
+    "We also limit our search by only looking for sequences that have IDSs created by a specific pipeline."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0c4fb0bc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "payload = {\n",
```

Comment:
Can `PIPELINE_THAT_CREATED_CHROMELEON_IDS` and `SEQUENCE_NAME` be made local vars, with markdown instructions on how to set them?

Also, this query fails if there are any results which don't have labels. Suggested fix: select only results with the `run_name` label in the ES request payload:

```
payload = {
    "size": 200,
    "query": {
      "bool": {
        "must": [
          {"term": {"idsType": "thermofisher-chromeleon"}},
          {"term": {"source.name": "PIPELINE_THAT_CREATED_CHROMELEON_IDS"}},
          {"wildcard": {"file.path": "*SEQUENCE_NAME.seq*"}},
          {
            "nested": {
              "path": "labels",
              "query": {
                "bool": {
                  "must": [
                    {"term": {"labels.name": "run_name"}}
                  ]
                }
              }
            }
          }
        ]
      }
    },
    "_source": [ "fileId", "filePath", "labels"]
}
```